src/Cake.SqlServer/Cake.SqlServer.nuspec

```diff
@@ -28,6 +28,8 @@
     <file src="net461/Cake.SqlServer.pdb" target="lib/net461" />
     <file src="net461/Cake.SqlServer.xml" target="lib/net461" />
 
+    <file src="net461/Azure.Core.dll" target="lib/net461" />

```

Comment:
Does that mean that we can connect to databases in Azure SQL behind 2FA??? i.e. DACFx lib supports it?

Reply:
More authentication options appeared in Microsoft.Data.SqlClient v3.0.0.
https://docs.microsoft.com/en-us/sql/connect/ado-net/sql/azure-active-directory-authentication?view=sql-server-ver15
 2FA - not sure how it could work